src/v0/destinations/tiktok_ads/transformV2.js

```diff
@@ -40,7 +40,7 @@ const getTrackResponsePayload = (message, destConfig, event) => {
   }
 
   // if contents is not present but we have properties.products present which has fields with superset of contents fields
-  if (payload.properties && !payload.properties.contents && message.properties.products) {
+  if (!payload.properties?.contents && message.properties?.products) {
```

Comment:
This is a code quality change and not functionality. Hence no difference should be there for the existing test cases.
Right ?
So why we have added test case?
Is this a part of oncall

Reply:
@Gauravudia is this a bugsnag issue? I mean where it broke?

Reply:
Yes, bugsnag issue, updated the PR description